doc/ale-json.txt

```diff
@@ -83,6 +83,40 @@ g:ale_json_fixjson_use_global                   *g:ale_json_fixjson_use_global*
   See |ale-integrations-local-executables|
 
 
+===============================================================================
+pytool                                                        *ale-json-pytool*
```

Comment:
I think you need to make sure this name matches the one in the ToC: `json.tool` or the linter tests won't pass.

Reply:
Ah, I forgot to add it to the toc!